cmd/vochaintest/vochaintest.go

```diff
@@ -192,6 +198,27 @@ func privKeyToSigner(key string) (*ethereum.SignKeys, error) {
 	return skey, nil
 }
 
+// testClient is a *client.Client but with additional methods
+// that are useful only in tests,
+// like waitUntilAccountExists or ensureProcessCreated
+type testClient struct {
+	*client.Client
+}
+
+// newTestClient will open a connection to addr.
+// If it fails, it will retry 10 times (with 1s interval)
+// before giving up
+func newTestClient(addr string) (*testClient, error) {
+	for tries := 10; tries > 0; tries-- {
```

Comment:
make this a constant or use the already defined "retries"

Reply:
ok, new constant